Modules presto-function-namespace-managers and presto-mysql failing in CI

https://github.com/prestodb/presto/actions/runs/12646508402/job/35238800130?pr=24326

```
2025-01-07T07:32:04.7779783Z [ERROR] Tests run: 91, Failures: 1, Errors: 0, Skipped: 75, Time elapsed: 1.312 s <<< FAILURE! - in TestSuite
2025-01-07T07:32:04.7795120Z [ERROR] com.facebook.presto.functionNamespace.mysql.TestMySqlFunctionNamespaceManager.setup  Time elapsed: 0.29 s  <<< FAILURE!
2025-01-07T07:32:04.7796290Z java.lang.RuntimeException: 
2025-01-07T07:32:04.7798483Z io.airlift.command.CommandFailedException: [/tmp/testing-mysql-server3402480835468141148/bin/mysqld, --no-defaults, --initialize-insecure, --skip-sync-frm, --innodb-flush-method=nosync, --datadir, /tmp/testing-mysql-server3402480835468141148/data] exited with 127
2025-01-07T07:32:04.7801460Z /tmp/testing-mysql-server3402480835468141148/bin/mysqld: error while loading shared libraries: libaio.so.1: cannot open shared object file: No such file or directory
2025-01-07T07:32:04.7802478Z 
2025-01-07T07:32:04.7803029Z 	at com.facebook.presto.testing.mysql.AbstractEmbeddedMySql.system(AbstractEmbeddedMySql.java:282)
2025-01-07T07:32:04.7804318Z 	at com.facebook.presto.testing.mysql.AbstractEmbeddedMySql.initialize(AbstractEmbeddedMySql.java:186)
2025-01-07T07:32:04.7805576Z 	at com.facebook.presto.testing.mysql.AbstractEmbeddedMySql.<init>(AbstractEmbeddedMySql.java:84)
2025-01-07T07:32:04.7806734Z 	at com.facebook.presto.testing.mysql.EmbeddedMySql5.<init>(EmbeddedMySql5.java:27)
2025-01-07T07:32:04.7807765Z 	at com.facebook.presto.testing.mysql.TestingMySqlServer.<init>(TestingMySqlServer.java:38)
2025-01-07T07:32:04.7808900Z 	at com.facebook.presto.testing.mysql.TestingMySqlServer.<init>(TestingMySqlServer.java:32)
2025-01-07T07:32:04.7810052Z 	at com.facebook.presto.testing.mysql.TestingMySqlServer.<init>(TestingMySqlServer.java:26)
2025-01-07T07:32:04.7811818Z 	at com.facebook.presto.functionNamespace.mysql.TestMySqlFunctionNamespaceManager.setup(TestMySqlFunctionNamespaceManager.java:98)
2025-01-07T07:32:04.7813149Z 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2025-01-07T07:32:04.7814053Z 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2025-01-07T07:32:04.7815159Z 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2025-01-07T07:32:04.7816101Z 	at java.lang.reflect.Method.invoke(Method.java:498)
2025-01-07T07:32:04.7817051Z 	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
2025-01-07T07:32:04.7818520Z 	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:65)
2025-01-07T07:32:04.7820030Z 	at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:381)
2025-01-07T07:32:04.7821760Z 	at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:319)
2025-01-07T07:32:04.7823271Z 	at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:178)
2025-01-07T07:32:04.7824523Z 	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:122)
2025-01-07T07:32:04.7825738Z 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
2025-01-07T07:32:04.7826913Z 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
2025-01-07T07:32:04.7827620Z 	at java.lang.Thread.run(Thread.java:750)
2025-01-07T07:32:04.7828862Z Caused by: io.airlift.command.CommandFailedException: [/tmp/testing-mysql-server3402480835468141148/bin/mysqld, --no-defaults, --initialize-insecure, --skip-sync-frm, --innodb-flush-method=nosync, --datadir, /tmp/testing-mysql-server3402480835468141148/data] exited with 127
2025-01-07T07:32:04.7830423Z /tmp/testing-mysql-server3402480835468141148/bin/mysqld: error while loading shared libraries: libaio.so.1: cannot open shared object file: No such file or directory
2025-01-07T07:32:04.7831052Z 
2025-01-07T07:32:04.7831518Z 	at io.airlift.command.ProcessCallable.call(ProcessCallable.java:64)
2025-01-07T07:32:04.7844411Z 	at io.airlift.command.ProcessCallable.call(ProcessCallable.java:22)
2025-01-07T07:32:04.7844858Z 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
2025-01-07T07:32:04.7845176Z 	... 3 more
2025-01-07T07:32:04.7845280Z 
2025-01-07T07:32:04.8017084Z [INFO] 
2025-01-07T07:32:04.8017393Z [INFO] Results:
2025-01-07T07:32:04.8017686Z [INFO] 
2025-01-07T07:32:04.8017948Z [ERROR] Failures: 
2025-01-07T07:32:04.8022550Z [ERROR]   TestMySqlFunctionNamespaceManager.setup:98 » Runtime io.airlift.command.CommandFailedException: [/tmp/testing-mysql-server3402480835468141148/bin/mysqld, --no-defaults, --initialize-insecure, --skip-sync-frm, --innodb-flush-method=nosync, --datadir, /tmp/testing-mysql-server3402480835468141148/data] exited with 127
2025-01-07T07:32:04.8024680Z /tmp/testing-mysql-server3402480835468141148/bin/mysqld: error while loading shared libraries: libaio.so.1: cannot open shared object file: No such file or directory
```

--- a/.github/workflows/test-other-modules.yml
+++ b/.github/workflows/test-other-modules.yml
@@ -53,6 +53,9 @@ jobs:
           key: ${{ runner.os }}-maven-2-${{ hashFiles('**/pom.xml') }}
           restore-keys: |
             ${{ runner.os }}-maven-2-
+      # Workaround for Ubuntu 24 and mysql to find the dependent library (https://github.com/prestodb/presto/issues/24327)
+      - name: Create symlink for libaio.so.1
+        run: sudo ln -s /usr/lib/x86_64-linux-gnu/libaio.so.1t64 /usr/lib/x86_64-linux-gnu/libaio.so.1
       - name: Populate maven cache
         if: steps.cache-maven.outputs.cache-hit != 'true'
         run: ./mvnw de.qaware.maven:go-offline-maven-plugin:resolve-dependencies --no-transfer-progress && .github/bin/download_nodejs

